internal/controller/api/v1alpha1/agent_types.go

```diff
@@ -133,6 +133,7 @@ type AgentSpec struct {
 	Enabled                 *bool             `json:"enabled,omitempty"`
 	IgnitionConfigOverrides string            `json:"ignitionConfigOverrides,omitempty"`
 	InstallerArgs           string            `json:"installerArgs,omitempty"`
+	Approved                bool              `json:"approved"`
```

Comment:
Not a blocker for this PR, but we don't have a description to any of the fields, maybe we should start adding description? 
as far as i know the comment above the filed will be used in the spec description of the CRD, @avishayt do you think we should start adding description?